book/website/welcome.md

```diff
@@ -1,13 +1,20 @@
 (welcome)=
 # Welcome
 
-_The Turing Way_ is a handbook to reproducible, ethical and collaborative data science. 
+*Welcome to The Turing Way guides to reproducible, ethical and collaborative data science.*
 
-The project is open source (anyone can [inspect the code](https://github.com/alan-turing-institute/the-turing-way)), open contribution (anyone can {ref}`contribute<ch-contributing>`), and open licenced (anyone can [re-use the materials](https://github.com/alan-turing-institute/the-turing-way/blob/master/LICENSE.md)).
+_The Turing Way_ is an open source, open collaboration and community-driven project.
```

Comment:
I am disappointed to see this revert to the previous form (after #1961).
In #1961 I expanded and clarified the terms, with the hope to make this sentence both easier to read and reduce jargon. 
I also flipped the sentence around so that we could avoid having to hyphenate open-source (https://github.com/alan-turing-institute/the-turing-way/pull/1961#issuecomment-884598869).

Reply:
Hey Danny, at the time of editing this, the branch was not rebased.
My aim was to come back and edit to make sure that I don't completely change your phrasing - while also not make the intro lines linking out. Please bare with me as I work through this.

Reply:
I can understand the desire to avoid links so early on.

I distinguish between open source (where you can *see* the source code) and open licenced (where you're allowed to *use* the source code), but perhaps that conceptual distinction is not one that actually occurs too much.

I think if there is room for 'community-driven' then it's a great addition - it is something which is important for the project that previously wasn't noted.

To fix the grammar it could be flipped around like so:
```suggestion
_The Turing Way_ project is open source, open collaboration, and community-driven.
```

_[I think I might have been tired/grumpy when I wrote the previous comment - I like this version better now than when I wrote my previous comment! :flushed:]_